openmc_plasma_source/ring_source.py

```diff
@@ -0,0 +1,45 @@
+
+import openmc
+
+
+class FusionRingSource(openmc.Source):
+    """An openmc.Source object with some presets to make it more convenient
+    for fusion simulations using a ring source. All attributes can be changed
+    after initialization if required. Default isotropic ring source with a Muir
+    energy distribution.
+
+    Args:
+        radius: the inner radius of the ring source
+        start_angle: the start angle of the ring in radians,
+        stop_angle: the end angle of the ring in radians,
+        temperature: the temperature to use in the Muir distribution in eV,
+    """
+    def __init__(
+        self,
+        radius,
+        start_angle: float =0.,
+        stop_angle: float = 6.28318530718,
+        temperature: float = 20000.,
```

Comment:
Should this be a user parameter? Do users always know the ion temperature or should it be abstracted too?

Reply:
I think some users will know the temperature and others won't. So for users who want to fine tune the temperature this is an option but for most who just want a typical temperature then there is a default value which is a regular fusion plasma temperature. The default temperature on openmc.Muir uses the same default temperature

Reply:
thanks for the 👍 , shall I resolved this one and the other similar review comments ?